pages/common/git-log.md

```diff
@@ -9,3 +9,11 @@
 - Show the history of a particular file or directory, including differences
 
 `git log -p PATH`
```

Comment:
since you're at it, it would probably make sense to convert `PATH` to the `{{token}}` format (lowercase and add the `{{}}`).